templates/security/disa-stig.html

```diff
@@ -66,13 +87,55 @@ <h2>The DISA-STIG guides for Ubuntu</h2>
          </td>
          <td>
            Defense Information System Agency <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">Security Technical Implementation Guides (STIGs)</a> for Ubuntu
-         </td>
-         <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
-         <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
+          </td>
+         <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/2ccda8d7-tick-orange.svg", alt="Yes: Configuration guide", width="14", height="14", hi_def=True, loading="lazy",) | safe }}</td>
+         <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/2ccda8d7-tick-orange.svg", alt="Yes: Configuration guide", width="14", height="14", hi_def=True, loading="lazy",) | safe }}</td>
          <td class="u-align--center"><i class="p-icon--success">Yes: Tooling and automation</i></td>
        </tr>
      </tbody>
    </table>
+   <ul class="p-inline-list" aria-label="key">
+    <li class="p-inline-list__item">
+      <img src="https://assets.ubuntu.com/v1/2ccda8d7-tick-orange.svg" alt="Yes: Configuration guide" style="height: 14px; vertical-align: middle; width: 14px;">
+      <small>Configuration guide</small>
+    </li>
+    <li class="p-inline-list__item">
+      <i class="p-icon--success">Yes</i>
+      <small>Tooling and automation</small>
+    </li>
+  </ul>
+  <a class="p-button--positive" href="/security/certification/docs/disa-stig">Read more about Ubuntu tooling and automation</a>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="u-sv3">How to audit and comply with DISA-STIG?</h2>
+      <ol class="p-stepped-list">
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Auditing</h3>
+          <div class="p-stepped-list__content">
+            <p>Using the Ubuntu Security Guide auditing is as simple as:</p>
+            <div class="p-code-snippet">
+              <pre class="p-code-snippet__block--icon"><code>sudo usg audit stig</code></pre>
+            </div>
+            <p>and get the report in machine readable or browser friendly format.</p>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Compliance</h3>
+          <div class="p-stepped-list__content">
+            <p>Using the Ubuntu Security Guide applying the necessary rules for compliance is as simple as:</p>
+            <div class="p-code-snippet">
+              <pre class="p-code-snippet__block--icon"><code>sudo usg fix stig</code></pre>
```

Comment:
can we change this to `disa_stig`?
The tooling changed since the page creation. I've updated the copydoc already